ui/apps/platform/src/Containers/Violations/ViolationsTableSearchFilter.tsx

```diff
@@ -12,6 +12,7 @@ import CompoundSearchFilter from 'Components/CompoundSearchFilter/components/Com
 import {
     Category as PolicyCategory,
     Name as PolicyName,
+    LifecycleStage as PolicyLifecycleStage,
```

Comment:
It might be worthwhile to export the attribute names like this so there aren't naming conflicts in files where we combine different types of attributes. So instead of exporting `Category` from `policy.ts`, we export `PolicyCategory`. Otherwise, we have to alias every attribute name. 

I'll consider doing that in a follow-up PR.